servers/plugins/org.fusesource.ide.server.fuse.core/src/org/fusesource/ide/server/fuse/core/server/subsystems/Fuse6xStartupLaunchConfigurator.java

```diff
@@ -14,49 +14,31 @@
 import org.eclipse.wst.server.core.IServer;
 import org.fusesource.ide.foundation.core.util.Strings;
 import org.fusesource.ide.server.fuse.core.util.FuseToolingConstants;
-import org.fusesource.ide.server.karaf.core.server.subsystems.BaseKarafStartupLaunchConfigurator;
+import org.fusesource.ide.server.karaf.core.server.subsystems.Karaf2xStartupLaunchConfigurator;
 
 /**
  * @author lhein
  */
-public class Fuse6xStartupLaunchConfigurator extends BaseKarafStartupLaunchConfigurator {
+public class Fuse6xStartupLaunchConfigurator extends Karaf2xStartupLaunchConfigurator {
```

Comment:
There is no advantage to inherit from Karaf2x class. Why did you change it?

Reply:
I think that it better reflect the reality (Fuse 6 is based on Karaf 2.x).